cli/sdk/package.json

```diff
@@ -41,6 +41,7 @@
     "@prisma/get-platform": "workspace:*",
     "archiver": "3.0.0",
     "chalk": "^3.0.0",
+    "checkpoint-client": "^1.0.7",
```

Comment:
Let's use the same version in both package.json's.
I suggest `^1.0.7` as we control that package and can trust the patches.

Reply:
👍